rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/cat.shards/10_basic.yml

```diff
@@ -1,8 +1,8 @@
 ---
 "Help":
   - requires:
-      cluster_features: [ "gte_v8.15.0" ]
-      reason: sparse vector was added in 8.15.0
+      cluster_features: [ "gte_v8.16.0" ]
+      reason: search idle state was added in 8.16.0
```

Comment:
I don't think this quite works unfortunately (the previous change of this nature looks incorrect to me). If we really want to proceed here we'll need to introduce a [new cluster feature](https://github.com/elastic/elasticsearch/blob/main/CONTRIBUTING.md#checking-for-cluster-features).

Reply:
I'm not sure I understand why this doesn't work? There's a few other files with similar formatting and some recent ones using `gte_v8.16.0` and [this section](https://github.com/elastic/elasticsearch/blob/main/rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/README.asciidoc#cluster_features) of the yaml docs suggests to me that it should be fine (and when I replace it with a garbage value the test fails, so it seems like it's reading it fine).